File: Java/Types of Matrix/Sum of upper and lower triangular elements of a matrix.java
class sum_upper_lower_triangular {
    public static void main(String[] args) {
        int[][] A = {
                { 1, 2, 3 },
                { 4, 5, 6 },
                { 7, 8, 9 }
        };
        System.out.println("Matrix: ");
        int Ut=0, Lt=0; 
        for (int row = 0; row < A.length; row++) {
            for (int col = 0; col < A[row].length; col++) {
                if(row <= col)
                    Ut += A[row][col];      // Upper Triangular
                if(row >= col)
                    Lt += A[row][col];      // Lower Triangular
                System.out.print("\t" + A[row][col]);
            }
            System.out.println();
        }
        System.out.println("\nSum of Upper Triangular = " + Ut);
        System.out.println("Sum of Lower Triangular = " + Lt);
    }
}

File: Python/Types of Matrix/Sum of upper and lower triangular elements of a matrix.py
# (3row X 3columns)
A = [[1, 2, 3],
     [4, 5, 6],
     [7, 8, 9]]

print("Matrix: ")
Ut=0; Lt=0 
for row in range(0, len(A)):
    for col in range(0, len(A[row])):
        if(row <= col):
            Ut += A[row][col]      # Upper Triangular
        if(row >= col):
            Lt += A[row][col]      # Lower Triangular
        print("\t", A[row][col], end='');  
    print()

print("\nSum of Upper Triangular =", Ut)
print("Sum of Lower Triangular =", Lt)